examples/by_feature/deepspeed_with_config_support.py

```diff
@@ -285,7 +285,7 @@ def evaluate(args, model, eval_dataloader, accelerator, eval_dataset):
             outputs = model(**batch)
 
         loss = outputs.loss
-        losses.append(accelerator.gather(loss.repeat(args.per_device_eval_batch_size)))
+        losses.append(accelerator.gather_for_metrics(loss.repeat(args.per_device_eval_batch_size)))
 
     losses = torch.cat(losses)
     losses = losses[: len(eval_dataset)]
```

Comment:
This line becomes redundant when using `gather_for_metrics`

Reply:
Ah yeah, makes sense thank you!